https://issues.redhat.com/browse/ACM-21153 (#7978)
open-cluster-management/rhacm-docs: business_continuity/backup_restore/backup_install.adoc

. Enable the `cluster-backup` option on the `MultiClusterHub` resource by setting `cluster-back` to `true`. 

[#installing-startingcsv]
=== Installing a _startingCSV_ on the OADP operator 

> Install the OADP operator at a specific version by setting the `startingCSV` property on the `installer.open-cluster-management.io/oadp-subscription-spec` annotation. After you enable the `cluster-backup` option on the `MultiClusterHub` resource, the OADP operator is installed by using the version specified by the `startingCSV` property.

Reviewer: ```suggestion
Install a specific version of the OADP operator by setting the `startingCSV` property on the `installer.open-cluster-management.io/oadp-subscription-spec` annotation. After you enable the `cluster-backup` option on the `MultiClusterHub` resource, the OADP operator is installed with the version that is specified by the `startingCSV` property.
```

Reviewer: at a specific version is odd here, we can make it more concise, I think.

Complete the following steps: 

. On your `MultiClusterHub` resource, set the following annotation, 

> +

Reviewer: please fix spacing, when these are not grayed out, remember that is the reason.

Author: Thanks, I did add a space, which I believe will fix it. However, both in the "view file" and adoc preview, it was grayed out with both the space added and not added, lol 

anyway, let me know if it looks good to you, thank you!

Reviewer: looks great to me!

. Enable the `cluster-backup` option on the `MultiClusterHub` resource by setting `cluster-back` to `true`. 

[#installing-startingcsv]
=== Installing a _startingCSV_ on the OADP operator 

> After you enable the `cluster-backup` option on the `MultiClusterHub` resource, you can set the `startingCSV` property to make it easier to store and transfer data across your hub clusters. 

Author: I believe I am right hat the `cluster-backup` must be enabled first. If I am wrong, please correct me.

> After you enable the `cluster-backup` option on the `MultiClusterHub` resource, you can set the `startingCSV` property to make it easier to store and transfer data across your hub clusters. 

Reviewer 2: No, this doesn't have to be enabled first; it's actually better to first set the annotation ( with the startingCSV property ) and after that enable the backup. Otherwise you install OADP at the latest version and then uninstall / install at the version requested by the annotation

Suggest this for line 115:

Install the OADP operator at a specific version by setting the `startingCSV` property on the `installer.open-cluster-management.io/oadp-subscription-spec` annotation.
After you enable the `cluster-backup` option on the `MultiClusterHub` resource, the OADP operator is installed using the version specified by the `startingCSV` property.

Complete the following steps: 

> . On your `MultiClusterHub` resource, set the following annotation, 

Author: There weren't many steps in the Jiras. Please correct me if I'm missing anything.

Reviewer: need a colon here?

+
[source,yaml]
----
installer.open-cluster-management.io/oadp-subscription-spec: ' {"source": <my-mirror-catalog-source>,"startingCSV": "<1>.v<2>"}
----

> <1> Your OADP operator. 

Reviewer 2: Should be : 

installer.open-cluster-management.io/oadp-subscription-spec: ' {"source": <my-mirror-catalog-source>,"startingCSV": "<1>"}

<1> The OADP operator version that will be installed from the specified source.

Author: @birsanv thanks! I applied all your feedback, but did want to avoid using future verb tense. Let me know what you think, thanks!
